sdk/iot/core/src/az_iot_sas_token.c

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#include <stdlib.h>
+#include <stdint.h>
+#include <az_iot_sas_token.h>
+#include <az_contract_internal.h>
+#include <az_span.h>
+#include <_az_cfg.h>
+
+#define LF                      '\n'
```

Comment:
It is an implementation file so things don't matter that much here, but in our code we'd sort includes this way:

```
#include <az_contract_internal.h>
#include <az_iot_sas_token.h>
#include <az_span.h>

#include <stdint.h>

#include <_az_cfg.h>
```

(or, 
```
#include <az_iot_sas_token.h>

#include <az_contract_internal.h>
#include <az_span.h>

#include <stdint.h>

#include <_az_cfg.h>
```
)

Reply:
Since the includes are not sorted, I'd recommend to run Ctrl+K+D (auto-format) on all the files that you're adding.

Reply:
Fixed.